state-lab-app/src/Components/Clicker.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+ class Clicker extends React.Component {
+  state = {
+    count: 0, 
+    multiplier: 1
+  }
+   handleClick = () => {
+    this.setState({
```

Comment:
You shouldn't update the state directly.

try using `this.setState((prevState)=> prevState.count + 1 * prevState.multiplier)`